src/main/java/org/folio/services/ledger/LedgerDetailsService.java

```diff
@@ -37,28 +40,31 @@ public Future<FiscalYear> getCurrentFiscalYear(String ledgerId, RequestContext r
   public Future<FiscalYear> getPlannedFiscalYear(String ledgerId, RequestContext requestContext) {
     return getFirstThreeFiscalYears(ledgerId, requestContext)
       .map(firstTwoFiscalYears -> {
-         FiscalYear curFY = defineCurrentFiscalYear(firstTwoFiscalYears);
-         int size = firstTwoFiscalYears.size();
-         int curIndex = firstTwoFiscalYears.indexOf(curFY);
-         int nextIndex = curIndex + 1;
-         if (curFY != null && nextIndex != size && size > 1) {
-           return firstTwoFiscalYears.get(curIndex + 1);
-         } else {
-           return null;
-         }
+        FiscalYear curFY = defineCurrentFiscalYear(firstTwoFiscalYears);
+        int size = firstTwoFiscalYears.size();
+        int curIndex = firstTwoFiscalYears.indexOf(curFY);
+        int nextIndex = curIndex + 1;
+        if (curFY != null && nextIndex != size && size > 1) {
```

Comment:
some info log can be added to this condition

Reply:
covered with logs